src/GitHub.InlineReviews/Services/PullRequestSession.cs

```diff
@@ -126,6 +117,34 @@ public string GetRelativePath(string path)
         }
 
         /// <inheritdoc/>
+        public async Task<IPullRequestReviewCommentModel> PostReviewComment(string body, string commitId, string path, int position)
```

Comment:
Should these methods be added as part of this PR or have they snuck in from somewhere else?